### PR TITLE
Add EasyCodingStandard

### LINKED ISSUE #19: Add EasyCodingStandard

### PR DESCRIPTION
**Pull request by TomasVotruba**
Closes #19 

Also standard namespace "Rector\*" is used, it always respect vendor first and package name second

- repository → namespace
- https://github.com/rectorphp/rector/ - `Rector\Rector`
- https://github.com/rectorphp/rector-ci/ - `Rector\RectorCI`

### CODE REVIEW

File: src/Controller/GitHubWebHookController.php [hunk 1]
```diff
@@ -15,6 +15,16 @@
...
 {
+    /**
+     * @var Client
+     */
+    private $client;
+
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
```
Comment (TomasVotruba) [inline src/Controller/GitHubWebHookController.php:26]: Here only `Client` was extracted, other changes were automated (mostly spacing, few type → $var naming)

File: src/HttpKernel/RectorCIKernel.php [hunk 2]
```diff
@@ -1,48 +1,59 @@
...
-        $routes->import($confDir.'/{routes}'.self::CONFIG_EXTS, '/', 'glob');
+        $confDir = $this->getProjectDir() . '/config';
+
+        $routeCollectionBuilder->import(
+            $confDir . '/{routes}/' . $this->environment . '/**/*' . self::CONFIG_EXTS,
+            '/',
+            'glob'
+        );
+        $routeCollectionBuilder->import($confDir . '/{routes}/*' . self::CONFIG_EXTS, '/', 'glob');
+        $routeCollectionBuilder->import($confDir . '/{routes}' . self::CONFIG_EXTS, '/', 'glob');
```
Comment (TomasVotruba) [inline src/HttpKernel/RectorCIKernel.php:57]: I hate this Kernel spam :angry: 

https://github.com/rectorphp/rector-ci/issues/27

Comment (JanMikes) [reply src/HttpKernel/RectorCIKernel.php:57]: Yeah 😄 i had just a quick start without proper project set up 😄

Comment (TomasVotruba) [reply src/HttpKernel/RectorCIKernel.php:57]: I know, it's not code review comment to your code.

It's just to release my frustration for this design decision in Symfony :D

Will fix in next PR

Comment (JanMikes) [reply src/HttpKernel/RectorCIKernel.php:57]: PRs are always welcomed, i would do it otherwise myself, but it was lower on priority list 😸

Comment (TomasVotruba) [reply src/HttpKernel/RectorCIKernel.php:57]: When someone appologize, I always remember this: https://www.boredpanda.com/stop-saying-sorry-say-thank-you-comic-yao-xiao :dog: 

Can I merge this?

Comment (JanMikes) [reply src/HttpKernel/RectorCIKernel.php:57]: Thank you ❤️ 

I will review right now

File: composer.json [hunk 3]
```diff
@@ -39,7 +39,8 @@
         "symfony/maker-bundle": "^1.0",
         "symfony/phpunit-bridge": "4.2.*",
         "symfony/profiler-pack": "*",
-        "symfony/web-server-bundle": "4.2.*"
+        "symfony/web-server-bundle": "4.2.*",
+        "symplify/easy-coding-standard": "^6.0"
```
Comment (JanMikes) [inline composer.json:43]: Shouldn't this be in dev dependencies?

File: config/services.yaml [hunk 4]
```diff
@@ -6,6 +6,8 @@ services:
         autowire: true      # Automatically injects dependencies in your services.
         autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
 
-    RectorCI\:
+    GuzzleHttp\Client: ~
```
Comment (JanMikes) [inline config/services.yaml:9]: Yeah i know what you wanted to do here, thank you, but i wont use guzzle at all, it was work in progress :-) though it is good for now.

**Review by JanMikes — APPROVED** [review]

File: composer.json (hunk 3 above)
Comment (JanMikes) [reply composer.json:43]: My bad, it is already there :-)

**Comment by TomasVotruba** [comment]
:clap:

**Pull request merged**